resources/smart-contracts-rust/post_create_command.py

```diff
@@ -0,0 +1,64 @@
+
+import json
+import subprocess
+from glob import glob
+from pathlib import Path
+
+
+def main():
+    prepare_vscode_settings()
+    prepare_gitignore()
+    add_examples_if_empty_workspace()
+
+
+def prepare_vscode_settings():
```

Comment:
So that the user doesn't have to run the command `MultiversX: Setup Workspace` in VSCode anymore.